### ISSUE TITLE
Missing quant files during tximport run

### ISSUE THREAD

**Issue by Miserlou** [AlexsLemonade/refinebio#1131]
```
data-refinery-log-group-circleci-prod log-stream-processor-docker-circleci-prod Traceback (most recent call last):
data-refinery-log-group-circleci-prod log-stream-processor-docker-circleci-prod   File "/home/user/data_refinery_workers/processors/utils.py", line 480, in run_pipeline
data-refinery-log-group-circleci-prod log-stream-processor-docker-circleci-prod     last_result = processor(last_result)
data-refinery-log-group-circleci-prod log-stream-processor-docker-circleci-prod   File "/home/user/data_refinery_workers/processors/salmon.py", line 740, in tximport
data-refinery-log-group-circleci-prod log-stream-processor-docker-circleci-prod     tximport_inputs = _get_tximport_inputs(job_context)
data-refinery-log-group-circleci-prod log-stream-processor-docker-circleci-prod   File "/home/user/data_refinery_workers/processors/salmon.py", line 729, in _get_tximport_inputs
data-refinery-log-group-circleci-prod log-stream-processor-docker-circleci-prod     quant_files.append(ComputedFile.objects.filter(result=result, filename="quant.sf")[0])
data-refinery-log-group-circleci-prod log-stream-processor-docker-circleci-prod   File "/usr/local/lib/python3.5/dist-packages/django/db/models/query.py", line 303, in __getitem__
data-refinery-log-group-circleci-prod log-stream-processor-docker-circleci-prod     return qs._result_cache[0]
data-refinery-log-group-circleci-prod log-stream-processor-docker-circleci-prod IndexError: list index out of range
```

**Comment by Miserlou** [comment]
@kurtwheeler if you have a db connection open can you see how often this one is occuring

**Comment by kurtwheeler** [comment]
```
data_refinery=> select count(*) from processor_jobs where pipeline_applied='SALMON' and failure_reason like '%list index out of range%' and end_time < now() - interval '1 days';
 count 
-------
   100
(1 row)
```

**Comment by kurtwheeler** [comment]
I suspect this may have been related to #2932. After we rerun the samples in that and cleanup the dead ones I'd be interested to see if these errors still persist.

**Comment by kurtwheeler** [comment]
This hasn't happened since `2019-03-14 21:32:08.214767+00`. I think this may be resolved by #2932.
